frontend/src/components/EmployeeCards/EmployeeCards.tsx

```diff
@@ -33,7 +33,12 @@ const EmployeeCard = ({
 }: EmployeeCardProps) => {
   const navigate = useNavigate();
   const netId = email.split('@')[0];
-  const fmtPhone = formatPhone(phoneNumber);
+  const fmtPhone = '';
+  if (phoneNumber !== undefined) {
+    const fmtPhone = formatPhone(phoneNumber);
+  } else {
+    const fmtPhone = '';
```

Comment:
I dont think this else statement is needed since you set fmtPhone to '' already on line 36


also, nit: try to keep all the logic for formatPhone within the helper function, ie. do the check for if the phone number is undefined within formatPhone itself, so that if the helper is used in the future, you don't have to repeat this logic

Reply:
Thanks, I'll fix it here. I left it in when debugging some error regarding workingContext sending duplicated request.